extra-utils/peazip/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=peazip
+PKGSEC=utils
+PKGDES="Cross-platform file and archive manager written in Pascal"
```

Comment:
No one cares about what language is written in. Leave out unnecessary detail in PKGDES.